Library/Formula/ipmitool.rb

```diff
@@ -2,18 +2,17 @@
 
 class Ipmitool < Formula
   homepage 'http://ipmitool.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/ipmitool/ipmitool/1.8.13/ipmitool-1.8.13.tar.bz2'
-  sha1 '22254a2b814c8cd323866a4dd835e390521c1dfa'
+  url 'https://downloads.sourceforge.net/project/ipmitool/ipmitool/1.8.15/ipmitool-1.8.15.tar.bz2'
+  sha1 '2c9c5d7c5a285586df508ad933577f275684353a'
+
+  depends_on "openssl"
 
   def install
     # tracking upstream: http://sourceforge.net/p/ipmitool/feature-requests/47/
     # fix build errors w/ clang
     inreplace 'include/ipmitool/ipmi_user.h', 'HAVE_PRAGMA_PACK', 'DISABLE_PRAGMA_PACK'
```

Comment:
@lloydde where is the commit you are referring to? Also, doesn't seem like there is a new release yet.